hack/mod-check

```diff
@@ -1,26 +1,26 @@
-#!/bin/bash
```

Comment:
Why was the mode of this file changed ? (would expect no changes at all here =P)
It is why the CI is broken: https://github.com/mineiros-io/terramate/runs/4686198413?check_suite_focus=true#step:4:6

Reply:
Hmm, i was not sure what was up with that either. Don't know why it changed.

Reply:
Odd indeed :thinking: ...maybe some vscode thing (I saw a .vscode being added on gitignore, so assuming vscode =P). In the end we need the original permissions restored.